add an admin guide for the rocur account (#62)
rladies/rladiesguide: content/rocur/admin guide/index.en.md

---
title: "Admin guide"
menuTitle: "Admin guide"
weight: 60
---

## Account Admin  

> The account admin for the rotating curator account are members of the R-Ladies community that keeps @WeAreRLadies running. The account admin could be a team of 1-4 people who can rotate responsibilities from month-to-month. Being an account admin is an opportunity to connect with R-Ladies from all over the world while facilitating an online community resource.     

Reviewer: ```suggestion
The account admin (team) for the rotating curator account are members of the R-Ladies community who keep @WeAreRLadies running. The account admin could be a team of 1-4 people who can rotate responsibilities from month-to-month. Being an account admin is an opportunity to connect with R-Ladies from all over the world while facilitating an online community resource.     
```

## Primary Responsibilities  

The responsibilities of the account admin are to:  
* Coordinate and grant Twitter account access to curators.
* Provide instructions to curators and answer questions as needed.  
* Recruit R-Ladies to be curators for the account.  
* Update and expand on the below procedures as needed.   

## Admin Procedures  

> ### When someone signs up to curate:    

Reviewer: Are there cases when someone signs up but their participation is declined?

Author: We haven't had a situation like this yet, but this may be possible in the future. Guidelines for this situation (and others) would be great to discuss as an admin team.

* Add them to the curator schedule as soon as possible so other potential curators know that week is unavailable.  
* The curator should send an email to WeAre@rladies.org with a photo for the account  

> * When this photo is received, save it in the Curator Photos Folder.  

Reviewer: Is this one some sort of shared drive?

Author: Yes! I have a shared google folder in the google drive of R-Ladies Chicago (I do not know if there is a better place for this at this time...)

  - Save it as **YYYY-MM-DD-firstname** (date is first day of scheduled curation week)  
* Reply to the curator’s email confirming their schedule week for curating. You may use the template below:  

>Hi NAME,
>
>Thank you for the photo. You are scheduled to curate the week of DATE. We will contact you 1 week before you are scheduled to curate with more details. Let us know if you have any questions.   
>Thanks!

### 1 Week before they are scheduled to curate:  

> * Create a draft of their infographic  

Reviewer: ```suggestion
* Create a draft of their infographic.  
```

How?

* Send an email to the curator that includes:  

>   - A reminder of when they are scheduled to curate  

Reviewer: ```suggestion
  - A reminder of when they are scheduled to curate;  
```

>   - A draft of the infographic, request that they approve or request changes to the draft  

Reviewer: ```suggestion
  - A draft of the infographic, request that they approve or request changes to the draft;  
```

>   - Link the Curator Guide for their reference. Ask them if they have any questions  

Reviewer: ```suggestion
  - Link the Curator Guide for their reference. Ask them if they have any questions.  
```

#### Sample email ~1 week before the first day of curating

>Hi **NAME**,
>
>We are excited to have you curate next week! You are scheduled to curate from Monday, **[DATE]** to Saturday **[DATE]**. Depending on your schedule, we can modify this slightly (i.e. start on Tuesday or end on Friday), just let us know. We appreciate your time and contributions to the R-Ladies RoCur!
>
>We’ve attached what we will post this Sunday to advertise your curation week. Please let us know any changes that you would like no later than **[DATE]**.
>
>Please read through our [Curator Guide](https://guide.rladies.org/rocur/guide/) in preparation for curating. Additionally, if you haven’t already, please join our R-Ladies Directory.
>

> >We will contact you later this week with more information about accessing our twitter account. In the meantime, let us know if you have any questions about curating.

Reviewer: ```suggestion
>We will contact you later this week with more information about accessing our Twitter account. In the meantime, let us know if you have any questions about curating.
```

>
>Thanks!

### Sunday before they are scheduled to curate:   

> * Post their final infographic to the RoCur feed (be sure to mention their personal twitter handle)  

Reviewer: ```suggestion
* Post their final infographic to the RoCur feed (be sure to mention their personal Twitter handle).  
```

> * Add the curator’s twitter handle as a contributor on the RoCur account (via Tweetdeck) and remove last week’s curator from the RoCur team  

Reviewer: ```suggestion
* Add the curator’s Twitter handle as a contributor on the RoCur account (via Tweetdeck) and remove last week’s curator from the RoCur team. 
```

> * Email the curator that they are now added to the twitter account.  

Reviewer: ```suggestion
* Email the curator that they are now added to the Twitter account.  
```

* They will need to visit [tweetdeck.twitter.com](https://tweetdeck.twitter.com/) while logged into their twitter account in order to tweet from the account.  

#### Same email ~1-2 days before the first day of curating

>Hi NAME,
>

> >We have added you as a contributor to the @WeAreRLadies account. While logged into twitter, visit: https://tweetdeck.twitter.com/ and you should see @WeAreRLadies under your “Accounts” (at the bottom left part of the menu). Twitter also has a tweetdeck guide that might be useful.  

Reviewer: link to tweetdeck guide?

>
>Please refer to our Curator Guide for our curator agreement and tips re: engaging with our Twitter audience. Your curatorship will start on Monday, at 7AM ET and end on Saturday, at 12PM ET.  
>
>If you have any questions or encounter any problem, you can send us an email at WeAre@rladies.org. Also, you can join our #rocur_twitter channel on the community slack where you can reach out to all RoCur Managers and get connected with past curators.  
>
>Thanks!  

## Recruiting Curators  

Account admin can recruit potential curators in a variety of ways:  

> * The R-Ladies Community Slack  

Reviewer: ```suggestion
* The R-Ladies Community Slack;  
```

> * Reaching out to organizers of different R-Ladies chapters   

Reviewer: ```suggestion
* Reaching out to organizers of different R-Ladies chapters;   
```

> * A general call on the @RLadiesGlobal or @WeAreRLadies Twitter handles  

Reviewer: ```suggestion
* A general call on the @RLadiesGlobal or @WeAreRLadies Twitter handles;  
```
